### DISCUSSION TITLE
why i get broken pipes (os error 32)?

### DISCUSSION THREAD

**Discussion by Fodesu** [rustdesk/rustdesk#4475] [Q&A]
client1: manjaro linux in vm
client2 : android
![image](https://github.com/rustdesk/rustdesk/assets/75713465/c8dc918c-c078-4883-8467-582535ec12db)

**Comment by Rack1538** [comment]
Same. I only get this on Android, not Windows.

**Reply by wade1444** [reply]
on android how to solve？

**Comment by da-the-dev** [comment]
Same thing on Fedora with an RPM package
